product/models.py

```diff
@@ -0,0 +1,108 @@
+from django.db import models
+
+class Category(models.Model):
+	name	= models.CharField(max_length	=45)
+
+	class Meta: 
+		db_table = 'categories'
+
+class Product(models.Model):
+	user          = models.ForeignKey('user.User', on_delete=models.CASCADE)
+	category		  = models.ForeignKey('Category', on_delete=models.CASCADE)
+	name          = models.CharField(max_length=45)
+	main_image    = models.DecimalField(max_digits=18, decimal_places=2)
+	money_replace = models.CharField(max_length=45)
+	create_at     = models.DateTimeField(auto_now_add=True)
+	update_at     = models.DateTimeField(auto_now=True)
+
+	class Meta: 
+		db_table = 'products'
+
+class Stock(models.Model):
+	product		= models.ForeignKey('Product', on_delete=models.CASCADE)
+	quantity	= models.IntegerField(default=0)
+
+	class Meta:
+		db_table = 'stocks'
+	
+class ProductTag(models.Model):
+	product	= models.ForeignKey('user.User', on_delete=models.CASCADE)
+	tag_id  = models.ForeignKey('Tag', on_delete=models.CASCADE)
```

Comment:
필드명을 ₩tag_id`로 선언하면, database에 `tag_id_id`로 생성됩니다.
그렇기 때문에 아래와 같이 수정해주세요~ 
```suggestion
	tag  = models.ForeignKey('Tag', on_delete=models.CASCADE)
```

Reply:
ManyToMany 관계는 Django의 ManyToManyField를 활용해서 만들어주세요~
[참고] https://docs.djangoproject.com/en/3.1/ref/models/fields/#manytomanyfield
![스크린샷 2021-01-13 오후 7 38 21](https://user-images.githubusercontent.com/53214510/104441355-e5b2e180-55d6-11eb-8354-8fe777a0af74.png)